sandbox/src/mocks/createReferralAndSendForTriage/responses/ReferralRequest.json

```diff
@@ -38,7 +38,21 @@
         "system": "https://fhir.nhs.uk/Id/ods-organization-code",
         "value": "R69"
       }
-    }
+    },
+    {
+	"url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ReferralState-1",
+	"extension": [
+           {
+		"url": "state",
+		"valueCoding": {
+			"system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralState-1",
+			"code": "NOT_BOOKED",
```

Comment:
I think the referral should be in a triage state here, perhaps `AWAITING_TRIAGE`. Would the API implementation also indicate the referral is in a triage state in this scenario?